tasks/main.yml

```diff
@@ -1,2 +1,209 @@
 ---
 
+- name: Install postfix
+  package:
+    name:
+      - postfix
+      - postfix-ldap
+      - postfix-pgsql
+    state: present
+
+- name: Setup LDAP authentication
+  block:
+    - name: Install saslauthd
+      package:
+        name:
+          - libsasl2-modules
+          - sasl2-bin
+        state: present
+
+    - name: Place LDAP connection config file
+      template:
+        src: saslauthd.conf.j2
+        dest: /etc/saslauthd.conf
+
+    - name: Place SMTP SASL config file
+      copy:
+        src: smtpd.conf
+        dest: /etc/postfix/sasl/smtpd.conf
+
+    - name: Place saslauthd-postfix config file
+      copy:
+        src: saslauthd-postfix
+        dest: /etc/default/saslauthd-postfix
+
+    - name: Create socket directory for saslauthd
+      shell: dpkg-statoverride --add root sasl 755 /var/spool/postfix/var/run/saslauthd
+      register: dpkg_sasl
+      changed_when: "'already exists' not in dpkg_sasl.stderr"
+      failed_when: "dpkg_sasl.rc != 0 and 'already exists; aborting' not in dpkg_sasl.stderr"
+
+    - name: Add group sasl to user postfix
+      user:
+        name: postfix
+        append: True
+        groups: sasl
+
+  when: postfix_sasl_enabled is defined
```

Comment:
Why work with is defined here and not with boolean values and a default of `false`?

(Same for other `_enabled` variables)

Reply:
Fixed.